### PR TITLE
[4.0] Azure cleanup

### PR DESCRIPTION
**Pull request by brianteeman**
As azure is removed from J4 this completes the removal of references to it

### CODE REVIEW

File: libraries/src/CMS/Schema/ChangeSet.php [hunk 1]
```diff
@@ -257,12 +257,6 @@ private function getUpdateFiles()
 		// Get the folder from the database name
 		$sqlFolder = $this->db->getServerType();
 
-		// For `mssql` server types, convert the type to `sqlazure`
```
Comment (mbabker) [inline libraries/src/CMS/Schema/ChangeSet.php]: We purposefully left this one here.  The actual database driver code is still here, we figured we could leave the schema adapter too.

Comment (brianteeman) [inline libraries/src/CMS/Schema/ChangeSet.php]: ok I have reverted that one

Comment (mbabker) [inline libraries/src/CMS/Schema/ChangeSet.php]: The test change too please?

Comment (brianteeman) [inline libraries/src/CMS/Schema/ChangeSet.php]: done as requested

**Comment by wilsonge** [comment]
Thankyou :)

**Pull request merged**